.travis.yml

```diff
@@ -1,15 +1,19 @@
+sudo: required
 language: python
 cache: pip
 addons:
   firefox: "49.0.1"
   apt:
     packages:
       - xvfb
+
 python:
   - 2.7
 services:
   - postgresql
   - neo4j
+  - docker
```

Comment:
Note: When I finish adding support for ECS in https://github.com/refinery-platform/django_docker_engine, the these tests could run either against that, or against the local docker engine like they do now.